frontend/src/components/RegisterForm.tsx

```diff
@@ -72,6 +72,23 @@ const RegisterForm = () => {
   const [showFormStatus, setShowFormStatus] = useState(false)
 
   const [registerUser] = useMutation(REGISTER)
+  const [loginUser] = useMutation(LOGIN)
```

Comment:
the register mutation returns the token from backend. Can't we just do something like to avoid duplicate code:

```
     const registerResponse = await registerUser({
        variables: {
          username: data.username,
          email: data.email,
          password: data.password,
        },
      })
      localStorage.setItem('token', registerResponse.data.register)
      window.location.href = '/'